services/core-web/src/components/common/RenderMultiSelect.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { FC, useRef } from "react";
```

Comment:
Would say just skip updating the core-web MultiSelect. (I deleted it on my branch already). Use common version instead.